packages/commonwealth/server/models/thread.ts

```diff
@@ -95,6 +108,7 @@ export default (
         allowNull: false,
       },
       snapshot_proposal: { type: dataTypes.STRING(48), allowNull: true },
+      inks: {type: dataTypes.JSONB, allowNull: true},
```

Comment:
`inks` => `links`
Also, I think we need some migration to create minimal dataset (or even empty tables with column). Right now, the app on the branch is not usable
![image](https://user-images.githubusercontent.com/14819225/233985752-7ba4006d-8391-4515-98cd-155d6ac1de12.png)

Reply:
Added the migration to add the column here from BE branch + fixed links